src/main/java/core/basesyntax/model/machine/Machine.java

```diff
@@ -1,6 +1,20 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "machines")
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 public class Machine {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
add getter and setter for this field
also, `GenerationType.IDENTITY` would not work for this inheritance type

Reply:
Done, generation type changed to TABLE